### PR TITLE
CVS-295: Add event handler to update in-flight licences with the details of a COM newly allocated to an offender

### PR DESCRIPTION
**Pull request by smcveigh941**

### CODE REVIEW

File: server/services/communityService.ts [hunk 1]
```diff
@@ -33,14 +34,18 @@ export default class CommunityService {
...
+  }
+
   async searchProbationers(searchCriteria: SearchDto): Promise<OffenderDetail[]> {
     return this.probationSearchApiClient.searchProbationer(searchCriteria)
   }
 
-  async getProbationer(nomisId: string): Promise<OffenderDetail> {
-    const deliusRecords = await this.probationSearchApiClient.searchProbationer({ nomsNumber: nomisId })
+  async getProbationer(searchDto: SearchDto): Promise<OffenderDetail> {
+    const deliusRecords = await this.probationSearchApiClient.searchProbationer(searchDto)
```
Comment (timharrison-moj) [inline server/services/communityService.ts:46]: Not sure why the change of parameter type? Do we search here by different values?

**Review by timharrison-moj — APPROVED** [review]
LGTM.

File: server/services/communityService.ts (hunk 1 above)
Comment (smcveigh941) [reply server/services/communityService.ts:46]: No reason, I was going to use the endpoint to search for offenders by CRN but didn't need to in the end
